saga_tools/slurm_run_python.py

```diff
@@ -68,16 +72,146 @@ def main(cluster_params: Parameters, job_param_file: Path) -> None:
 
 
 @attrs(frozen=True, slots=True, kw_only=True)
-class SlurmPythonRunner:
+class SpackPackage:
+    package_name: str = attrib(validator=instance_of(str))
+    version: str = attrib(validator=instance_of(str))
```

Comment:
The full specification of a package is more complicated than just an alphanumeric string after the `@`, but I doubt we'll ever run into that.

Reply:
Right - we can always make it more complex in the future if someone needs it.